oxidation/bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -101,15 +101,18 @@ JsFunction
 {%- elif type.kind == "str" -%}
 {{ js_val }}.value({{ mut_cx_ref }})
 {%- elif type.kind == "str_based" -%}
-{% if type.custom_from_rs_string -%}
-{let custom_from_rs_string = {{ type.custom_from_rs_string }};
-match custom_from_rs_string({{ js_val }}.value({{ mut_cx_ref }}))
-{%- else -%}
-{match {{ js_val }}.value({{ mut_cx_ref }}).parse()
-{%- endif %} {
-    Ok(val) => val,
-    Err(err) => return cx.throw_type_error(err),
-}}
+{
+    {% if type.custom_from_rs_string -%}
+    let custom_from_rs_string = {{ type.custom_from_rs_string }};
```

Comment:
see https://github.com/Scille/parsec-cloud/pull/4245#discussion_r1141785242

Reply:
The comment in question:

> We could however add typing info in the `custom_from/to_rs_string`:
> 
> ```rust
>     let custom_from_rs_string: Fn(String) -> Result<{{ type.rs_type }}, AsRef<str>> = {{ type.custom_from_rs_string }};
>     let value: Result<{{ type.rs_type }}, _> = custom_from_rs_string({{ js_val }}.value({{ mut_cx_ref }}));
> ```


1. For `Fn(String) -> Result<{{ type.rs_type }}, AsRef<str>>`: Would not work => `AsRef<str>` is a trait not a concrete type (spoiler using `impl` won't work)
2. Could do

Reply:
We already type the return value on the `custom_{from,to}_rs_string`.
Is this really useful ?

Reply:
This PR is blocking the progress of other, so I merge it.
For you recommendation I've opened the issue #4251

Reply:
> We already type the return value on the custom_{from,to}_rs_string.

The idea is to help when we write the string containing the custom type, it makes it obvious when the signature of the function should be (instead of relying on copy/paste from other strings)

This is just a nice-to-have, as we don't add custom strings that often ;-)